pkg/policy/rule.go

```diff
@@ -183,11 +232,15 @@ func (r *rule) canReach(ctx *SearchContext, state *traceState) api.Decision {
 		for _, sel := range r.FromEndpoints {
 			ctx.PolicyTrace("    Allows from labels %+v", sel)
 			if sel.Matches(ctx.From) {
-				ctx.PolicyTrace("+     Found all required labels\n")
-				return api.Allowed
+				ctx.PolicyTrace("      Found all required labels")
+				if len(r.ToPorts) == 0 {
+					ctx.PolicyTrace("+       No L4 restrictions; allowing\n")
+					return api.Allowed
+				}
+				ctx.PolicyTrace("        Restricted to specific L4 destinations; skip during Label stage\n")
```

Comment:
Any preferences on how this label should indicate that the label stage cannot select this rule because it includes L4 requirements? Related: #1610

Reply:
maybe specify how you cannot make a purely L3 / label-specific decision whether policy allows or denies? E.g.:
```
L3 traffic restricted to specific L4 destinations; cannot determine whether policy allows traffic purely based off of L3
```

Reply:
How about this, a little more terse: 
```Rule restricts traffic to specific L4 destinations; deferring policy decision to L4 policy stage```